src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -246,7 +246,9 @@ def start_transfer(transfer_name, transfer_type, accession, paths, row_ids):
 
         if helpers.file_is_an_archive(path):
             transfer_dir = temp_dir
-            filepath = os.path.join(temp_dir, os.path.basename(path))
+            p = path.split(':',1)[1]
```

Comment:
thanks good catch, I am doing this on a temporary machine, not usual dev environment.